src/utils/index.ts

```diff
@@ -9,9 +9,13 @@ export function renderConfigToUrlParams(config: StacRenderObject): string {
   for (const [key, value] of Object.entries(params)) {
     if (!value) continue;
 
-    if (Array.isArray(value)) {
+    if (Array.isArray(value) && key !== 'colormap') {
       queryObj[key] = value.join(',');
-    } else {
+    } else if(key === 'colormap') {
```

Comment:
Is `colormap` being used? I can only find `colormap_name` in the collection specs.

Reply:
which collection specs are you referring to? There is a colormap being used for the `REFC__entire_atmosphere__*` object in the renders object of https://stac.eoapi.dev/collections/noaa-hrrr-sfc-conus